src/app/results/results.component.ts

```diff
@@ -212,6 +212,9 @@ export class ResultsComponent implements OnInit {
       urldata.rows = Number(query['rows']) || 10;
       this.presentPage = Math.abs(query['start'] / urldata.rows) + 1;
       let querydata = Object.assign({}, urldata);
+      if (this.resultDisplay !== 'news') {
```

Comment:
Please explain why are you doing this. It seems a mere workaround.

Reply:
Actually it is not a workaround, it is a fix. If we are not in news tab we will replace ` site:www.dailymail.co.uk` to ''
` site:` is used after a query to filter all results from that site.

Reply:
Can you create an action and effect to keep news search independent from normal search queries? In this way you would not need to replace the site query explicitly and both the searching will be independent and each time user switches from news tab and other tabs, it won't affect other.

Reply:
Yes I am working on it but till then it will be a fix.

Reply:
Please review the changes and tell if it has any limitation or any problem it causes.